module/VuFind/src/VuFind/ILS/Driver/Alma.php

```diff
@@ -464,7 +465,7 @@ public function checkRequestIsValid($id, $data, $patron)
         if ('copy' === $level) {
             // Call the request-options API for the logged-in user
             $requestOptionsPath = '/bibs/' . rawurlencode($id)
-                . '/holdings/' . rawurlencode($data['holding_id']) . '/items/'
+                . '/holdings/' . rawurlencode($data['holdings_id']) . '/items/'
```

Comment:
I think we need `$data['holdings_id'] ?? $data['holding_id']` here to provide back-compatibility for existing Alma ini files that have holding_id in HMACKeys.

Reply:
Good point. I think there are actually two places where we need to do this. I've added it in both, along with comments.

Reply:
True, I didn't get far enough to see that. The latter fallback has wrong variable name, though.

Reply:
Early morning copy/paste strikes again! Fortunately, continuous integration caught this, so our systems are working. :-) Fixed now!

Reply:
Thanks, all good now!

Reply:
Thanks!